lib/internal/http2/core.js

```diff
@@ -1609,6 +1609,9 @@ class Http2Stream extends Duplex {
       return;
     }
 
+    if ((typeof data === 'string' || Buffer.isBuffer(data)) && !data.length)
+      return;
```

Comment:
The first piece of the conditional is always true; Also, you’d still need to call `cb`.

Reply:
I'm confused that is the logic below the same?

```js
req.write('');
req.end();
```

vs

```js
req.end();
```

Reply:
That means can I actully ignore empty string stream?